src/main/java/io/confluent/idesidecar/restapi/clients/ClientConfigurator.java

```diff
@@ -318,18 +318,10 @@ public static Map<String, Object> getSchemaRegistryClientConfig(
       props.put("schema.registry.request.timeout.ms", defaultTimeout.toMillis());
     }
 
-    // CCloud requires the logical cluster ID to be set in the properties, so examine the URL
-    var logicalId = CCloud.SchemaRegistryIdentifier
-        .parse(srUri)
-        .map(CCloud.LsrcId.class::cast)
-        .map(CCloud.LsrcId::id)
-        .orElse(null);
-
     // Add any properties for SR credentials (if defined)
     var options = connection
         .getSchemaRegistryOptions()
-        .withRedact(redact)
-        .withLogicalClusterId(logicalId);
+        .withRedact(redact);
```

Comment:
The Confluent Cloud logical cluster id for a schema registry will need to be provided by the user as part of the credentials config in case of OAuth credentials. We cannot infer the "logical" cluster id from the Schema Registry URI which may or may not contain the cluster id in its first part. Moreover, a given physical cluster id may host multiple logical cluster ids.